ca_node/scripts/smach_ranking_controller.py

```diff
@@ -4,40 +4,54 @@
 import smach
 import smach_ros
 
-from ca_msgs.msg import Bumper
+from ca_msgs.msg import Bumper, Cliff
 from geometry_msgs.msg import Twist
 
 is_left_pressed = False
 is_right_pressed = False
+is_cliff_left = False
```

Comment:
Instead of declaring a lot of variables here. Define a [Python map](https://www.geeksforgeeks.org/python-map-function/) where the keys are side_left, front_left, etc, and the values are booleans.

Reply:
I'm not sure how to do that, since when I create the `list` of values to put them inside the `map` function, it says that they are not defined, in the examples that I saw the members of the list are `const`, not an undeclared variable.

Reply:
Check [this](https://repl.it/@ithenrik/Python-maps) example.